master/static/migrations/20230612112824_add-rp-workspace-bindings.tx.up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE rp_workspace_bindings ( 
+    workspace_id INT NOT NULL, 
+    pool_name TEXT NOT NULL, 
+    validity BOOLEAN NOT NULL,   
+    FOREIGN KEY (workspace_id) REFERENCES workspaces(id) ON DELETE CASCADE 
+);
+
+ALTER TABLE workspaces 
+    ADD COLUMN default_pool TEXT DEFAULT NULL;
```

Comment:
will workspaces have a default compute and aux pool?

Reply:
good point, I'll ask Haran but rn seems like it's just for compute and otherwise would use the cluster default aux pool